includes/GlobalNewFilesHooks.php

```diff
@@ -43,8 +43,12 @@ public static function onCreateWikiStatePublic( $dbname ) {
 	}
 
 	public static function onUploadComplete( $uploadBase ) {
-		MediaWikiServices::getInstance()->getJobQueueGroup()->push(
-			new GlobalNewFilesInsertJob( $uploadBase->getTitle(), [] )
+		$user = RequestContext::getMain()->getUser();
+		$services = MediaWikiServices::getInstance();
+		$centralIdLookup = $services->getCentralIdLookup();
+		$userId = $centralIdLookup->centralIdFromLocalUser( $user );
```

Comment:
Technically could all go in one expression, but it's minor enough that I don't care